conanfile.py

```diff
@@ -81,7 +81,7 @@ def configure(self):
             self.options.rm_safe("fPIC")
 
     def requirements(self):
-        self.requires("boost/1.78.0")
+        self.requires("boost/1.83.0")
```

Comment:
try boost/1.82.0?

Reply:
boost 1.82.0 works as well. Alternatively, milvus can go to 1.83.0. Personally, I see no reason to stay with 1.82.0 if 1.83.0 is already out